src/smart-components/portfolio/share-portfolio-modal.js

```diff
@@ -17,7 +24,11 @@ import useQuery from '../../utilities/use-query';
 import useEnhancedHistory from '../../utilities/use-enhanced-history';
 import { UnauthorizedRedirect } from '../error-pages/error-redirects';
 
-const SharePortfolioModal = ({ closeUrl, removeQuery }) => {
+const SharePortfolioModal = ({
+  closeUrl,
+  removeQuery,
+  portfolioName = () => ''
```

Comment:
Can we just pull in the portfolio name from the store with selector?

Reply:
that will work from the portfolio details page, but not from the kebab menu on the portfolio card

Reply:
@Hyperkid123 - that will work from the portfolio details page, but not from the kebab menu on the portfolio card

Reply:
oh yeah there is multiple entry points